src/test/scala/temple/utils/FileUtilsTest.scala

```diff
@@ -0,0 +1,15 @@
+package temple.utils
+
+import java.nio.file.FileAlreadyExistsException
+
+import org.scalatest.{FlatSpec, Matchers}
+
+class FileUtilsTest extends FlatSpec with Matchers {
+  "Creating a folder that already exists" should "not throw an exception" in {
+    FileUtils.createDirectory("src")
```

Comment:
Can we do this twice, to make it more explicit that the `src` directory exists? Could even wrap the second in `noException should be thrownBy`